keyboards/1upkeyboards/1up60hse/keymaps/via/config.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2018 MechMerlin
+ * Copyright 2018 Logan Huskins
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// place overrides here
+#define LAYER_STATE_8BIT
```

Comment:
I don't see the point of this if `MOUSEKEY=no` is in `rules.mk`. Prefer not to create yet another thing people copy blindly when adding VIA support.

Reply:
I must have misunderstood your comment on https://github.com/qmk/qmk_firmware/pull/7960#issuecomment-577465358

Reply:
There are a number of reasons to have this, actually. 

Not only does it reduce compile size, but it may speed up the checks for the keycodes, since there are fewer layers to iterate from.  Which given the fact that it's querying EEPROM for the layouts, may be a valuable thing.

Reply:
drashna, I would doubt this is a very big performance hit in the iteration. Also its only extreme cases where people have more than 4.

Reply:
But still gentlemen, I need to know how to proceed with this =).

Reply:
I agree, performance difference is negligible, obviously it isn't querying keycodes out of the keymap for undefined layers. Keep it simple.

Reply:
@mechmerlin I'd agree with removing the `LAYER_STATE_8BIT` definition, personally.

Reply:
Thanks guys, I'll take it out =)